include/deal.II/base/config.h.in

```diff
@@ -267,6 +267,23 @@
 
 
 /***********************************************************************
+ * A macro that is used to mark function as not throwing any exceptions.
+ * The appropriate way to do this in C++98 was to say
+ *     void foo () throw ();
```

Comment:
Correct. What's in the parentheses is a *list* of possible exception types. If the list is empty, then the function is not allowed to throw.